app/views/categories/show.html.erb

```diff
@@ -25,6 +25,6 @@
     </li>
   <% end %>
 </ul>
-<div class="my-3">
+<div class="mb-3 mt-auto">
   <%= link_to 'Add new expense', new_user_expense_path(current_user), class: 'btn btn-secondary w-100' %>
 </div>
```

Comment:
- [OPTIONAL] Great work making the changes to the **_Add New Expense_** button as requested by the previous reviewer. However, I strongly advise that you make the `background-color` of the button `5fb523` and not the ash-like colour that it has. Also, please work on the padding and the inner white border line as requested by the previous reviewer. 👍 😎 

### Your Design

![image](https://github.com/M-Anwar-Hussaini/Expense-Management/assets/77106962/85024e6c-3722-4a57-9dfc-14a1fd116ebc)

### Behance Design

![image](https://github.com/M-Anwar-Hussaini/Expense-Management/assets/77106962/6dbcfa5f-d2d4-4b51-a5fc-ffc5f8959796)